src/cephadm/cephadmlib/container_types.py

```diff
@@ -8,7 +8,29 @@
 from typing import Dict, List, Optional, Any, Union, Tuple, Iterable, cast
 
 from .call_wrappers import call, call_throws, CallVerbosity
-from .constants import DEFAULT_TIMEOUT
+from .constants import (
```

Comment:
`check-black` failed saying
```
would reformat /home/jenkins-build/build/workspace/ceph-pull-requests/src/cephadm/cephadmlib/container_types.py
```
I think you need to run `tox -e format-black` within src/cephadm to get it formatted correctly. That's why make check failed.

Reply:
Thank you! I have updated the PR.